redisinsight/api/src/modules/database/providers/database-info.provider.ts

```diff
@@ -177,6 +180,20 @@ export class DatabaseInfoProvider {
           parseInt(get(memoryInfo, 'number_of_cached_scripts'), 10) ||
           undefined,
         server: serverInfo,
+        stats: {
+          instantaneous_ops_per_sec:
+            get(statsInfo, 'instantaneous_ops_per_sec') || undefined,
```

Comment:
there is no need for `|| undefined`
as I remember lodash `get` function has 3d argument which is `default` value which is  already `undefined` here `get(statsInfo, 'instantaneous_ops_per_sec')`